JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -260,6 +265,18 @@ fun sharePost(post: Post, context: Context) {
     context.startActivity(Intent.createChooser(intent, context.getString(R.string.article_share_post)))
 }
 
+/**
+ * Opens chat for the article
+ *
+ * @param post to grab id from
+ * @param context Android context to show the chat in
+ */
+fun openChat(post:Post, context: Context) {
```

Comment:
Code format, please check it for the modified files:
`post:Post`, missing whitespace after semicolon